packages/ssx-server/src/server.ts

```
@@ -133,13 +129,15 @@ export class SSXServer extends EventEmitter {
       promises.push(this.resolveEns(siweMessage.address, resolveEnsOpts));
     }
     try {
-      siweMessageVerifyPromise = await Promise.all(promises)
-        .then(([siweMessageVerify, ensData]) => {
-          ens = ensData
+      siweMessageVerifyPromise = await Promise.all(promises).then(
+        ([siweMessageVerify, ensData]) => {
+          ens = ensData;
           return siweMessageVerify;
-        });
+        },
+      );
     } catch (error) {
       console.error(error);
+      throw error;
```

Do we want to throw an error? We are returning the state just below